private/scripts/helpers.sh

```diff
@@ -230,13 +264,14 @@ function update_php() {
     # Test for PHP version declartion already in pantheon.yml.
     if [ ! "$phpDeclaredInFile" -eq 0 ]; then
       # Update version to 8.x.
-      sed -i '' "s/7.4/8.0/" pantheon.yml
+      sed -i '' "s/7.4/${phpversion}/" pantheon.yml
     else
       # Add full PHP version declaration to pantheon.yml.
-      echo "php_version: 8.0" >> pantheon.yml
+      echo "php_version: ${phpversion}" >> pantheon.yml
     fi
-    git commit -am "[Sage Install] Update PHP version to 8.0"
-    git push origin master
+    git status
```

Comment:
```suggestion
```
still useful?

Reply:
No because what I actually wanted was to see what version actually got added to the `pantheon.yml`. What I actually want is a `git diff`.